jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -2330,6 +2330,16 @@ properties:
             type: string
             enum: [exptime, ivm]
             fits_keyword: RESWHT
+          pixfrac:
+            title: Drizzle parameter describing pixel shrinking
+            type: number
+            fits_keyword: PIXFRAC
+            fits_hdu: SCI
```

Comment:
Shouldn't these be in the primary header now, cuz they're aren't grouped with the photometry keywords anymore.

Reply:
Done